lib/components/javascript.js

```diff
@@ -7,7 +7,7 @@ const proxyAccessFunction = function (fqParts, opts = {}) {
     return new Proxy(target, {
         get: function (target, prop) {
             if (cds.entities) {
-                target.__proto__ = cds.entities(fqParts[0], fqParts[1])
+                target.__proto__ = cds.entities(fqParts[0])[fqParts[1]]
```

Comment:
is there a difference between `cds.entities('namespace.Entity')` and `cds.entities('namespace')('Entity')`. If not, should we not just leave them concatenated in the first place?

Reply:
Yes, there is a difference. `cds.entities(namespace)` will only return the children of a namespace. So, if you pass already the fqn of an entity, you won't get anything back.
If we would define an inline composition on an Entity, then the fqn of the entity is a new namespace.
e.g.

```cds
namespace my.bookshop;

entity Books : managed, cuid {
  title       : String  @title: 'Title';
  stock       : Integer @title: 'Stock';
  publishedOn : Date    @title: 'Published On';
  authors       : Composition of many {
                  key name : String;
                }
}
```

then `cds.entities('my.bookshop.Books')` would give us access to the `authors` entity.